### PR TITLE
Add rpm installation method

### PR DESCRIPTION
**Pull request by peng9808**
**Update the process of DRBD installation:**

Check if offline packages already exist before compiling drbd packages.

**Benfits:**

This change will reduce the pre-installation steps required for users to install DRBD. If the customer's operating system and kernel version are within the supported range, they will not need to install the kernel-devel package separately.

**Notes:**

Afterwards, we will update more validated offline installation packages, including Ubuntu, CentOS, Kylin, and other operating systems.  For specific version information, you can click on https://hwameistor.io/docs/install/prereq.

### CODE REVIEW

File: docker-drbd9/entry.sh [hunk 1]
```diff
@@ -224,6 +225,46 @@ modprobe_deps() {
 
 ### main
 modprobe_deps
+
```
[the remark this answers is not in the document]
Comment (peng9808) [reply docker-drbd9/entry.sh:239]: same as above

File: docker-drbd9/entry.sh [hunk 2]
```diff
@@ -224,6 +225,46 @@ modprobe_deps() {
...
+if [ "$substr" != "deb" ]; then
+    if [ -z "$OS_KERNEL" ]; then
+      echo "The system parameters are obtained successfully （OS_KERNEL）"
+      exit 1
+    fi
+
+    drbd_rpm=$(find "$RPMDIR" -type f -name "*$OS_KERNEL*" -print -quit)
+    utils_rpm=$(find "$RPMDIR" -type f -name "drbd-utils*" -print -quit)
+    if [ -z "$drbd_rpm" ]; then
+      echo "There is no corresponding kernel version rpm package"
```
[the remark this answers is not in the document]
Comment (peng9808) [reply docker-drbd9/entry.sh:251]: already edited

File: docker-drbd9/entry.sh [hunk 3]
```diff
@@ -224,6 +225,46 @@ modprobe_deps() {
 
 ### main
 modprobe_deps
+
+result=$(lbdisttool.py --os-release $HOSTRELEASE -l || echo "")
+substr="${result:0:3}"
+if [ "$substr" != "deb" ]; then
+    if [ -z "$OS_KERNEL" ]; then
+      echo "The system parameters are obtained successfully （OS_KERNEL）"
```
[the remark this answers is not in the document]
Comment (peng9808) [reply docker-drbd9/entry.sh]: already edited

**Review by SSmallMonster — APPROVED** [review]

**Comment by SSmallMonster** [comment]
lgtm

**Pull request merged**